docs/products/flink/howto/create-flink-applications.rst

```diff
@@ -0,0 +1,58 @@
+Create an Aiven for Apache Flink® application 
+=============================================
+
+:doc:`Aiven for Flink applications <../concepts/flink-applications>` in Aiven for Apache Flink® servers as a container that includes everything connected to a Flink job, including source and sink connections and data processing logic. The `Aiven Console <https://console.aiven.io/>`_ provides a guided wizard to help you build and deploy applications, simplifying the process of selecting the source and sink tables, writing data transformation statements, and validating and ingesting data through the interactive query feature.
+
+This article provides the information required to build and deploy applications on Aiven for Apache Flink service. 
+
+Create an application via the Aiven console
+--------------------------------------------
+
+Follow these steps to build your first Aiven for Flink application: 
```

Comment:
Should we have a warning that people need to create an integration first?

Reply:
I wanted to add this point as a prerequisite, and it looks like I missed it. Thanks for this.